gitlab/v4/cli.py

```diff
@@ -298,13 +304,17 @@ def _populate_sub_parser_by_class(
     if mgr_cls.__name__ in cli.custom_actions:
         name = mgr_cls.__name__
         for action_name in cli.custom_actions[name]:
-            sub_parser_action = sub_parser.add_parser(action_name)
+            sub_parser_action = action_parsers.get(action_name)
+            if sub_parser_action is None:
+                sub_parser_action = sub_parser.add_parser(action_name)
```

Comment:
I'm thinking the `get()` can already handle `None` values on its own and might be more efficient, though not sure how black will behave here:

```suggestion
            sub_parser_action = action_parsers.get(action_name, default=sub_parser.add_parser(action_name))
```
(Same with the other one)

Reply:
I thought that same thing. But unfortunately the function in the "default" value always gets called even if the value is not used. This breaks things 😢

```
$ cat test.py
#!/usr/bin/python3 -ttu
def hello():
    print("hello")
a = {"1": "test"}
print(a.get("1", hello()))
```

Run it

```
$ python test.py
hello
test
```